docs/Analyzer Configuration.md

```diff
@@ -654,17 +656,17 @@ Configurable Rules: [CA1010](https://docs.microsoft.com/dotnet/fundamentals/code
 Option Values: List (separated by '|') of interface names with their required generic fully qualified interface (separated by '->').
 Allowed interface formats:
 
-  1. Interface name only (includes all interfaces with the name, regardless of the containing type or namespace)
-  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format) with an optional "T:" prefix.
+  1. Interface name only (includes all interfaces with the name, regardless of the containing type or namespace).
+  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format) with an optional `T:` prefix.
 
-Default Value: None
+Default Value: _None_
 
 Examples:
 
 | Option Value | Summary |
 | --- | --- |
-|`dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`\``1` | All types implementing 'ISomething' regardless of its namespace are expected to also implement 'System.Collections.Generic.IEnumerable`1'. |
-|`dotnet_code_quality.CA1010.additional_required_generic_interfaces = T:System.Collections.IDictionary->T:System.Collections.Generic.IDictionary`\``2` | All types implementing 'System.Collections.Generic.IDictionary' are expected to also implement 'System.Collections.Generic.IDictionary`2'. |
+| ``dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`1`` | All types implementing 'ISomething' regardless of its namespace are expected to also implement 'System.Collections.Generic.IEnumerable\`1'. |
+| ``dotnet_code_quality.CA1010.additional_required_generic_interfaces = T:System.Collections.IDictionary->T:System.Collections.Generic.IDictionary`2`` | All types implementing 'System.Collections.Generic.IDictionary' are expected to also implement 'System.Collections.Generic.IDictionary`2'. |
```

Comment:
```suggestion
| ``dotnet_code_quality.CA1010.additional_required_generic_interfaces = T:System.Collections.IDictionary->T:System.Collections.Generic.IDictionary`2`` | All types implementing 'System.Collections.Generic.IDictionary' are expected to also implement ``System.Collections.Generic.IDictionary`2``. |
```

Reply:
This one is expected to not render correctly before the fix. The backtick wasn't escaped, nor included between double backticks.

Reply:
We have to decide if for the description we want to keep single quote (in which case the right fix is "\`") or if we want to go for "code" in which case your solution is good.

Reply:
The docs team always goes for code for any code element name.

btw, I think there is a typo in this sentence, there is an extra "Generic" in the namespace for non-generic `IDictionary` interface:

> All types implementing 'System.Collections.~~Generic.~~IDictionary'

@Evangelink Can you confirm?

Reply:
Namespace does include `Generic`, see https://docs.microsoft.com/dotnet/api/system.collections.generic.idictionary-2

Reply:
@Evangelink There are two `IDictionary` interfaces being referred to here. The generic one which has a `2 at the end, and the non-generic one:

https://docs.microsoft.com/en-us/dotnet/api/system.collections.idictionary?view=net-5.0

Reply:
Oh sorry I was looking only at the second part!